### ISSUE TITLE
Model files not found but I've copied it to root directory

### ISSUE THREAD

**Issue by proVoice228** [CorentinJ/Real-Time-Voice-Cloning#476]
2020-08-08 13:16:05.767958: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'cudart64_100.dll'; dlerror: cudart64_100.dll not found
2020-08-08 13:16:05.768070: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
C:\Users\Андрей\Desktop\Real-Time-Voice-Cloning-master\encoder\audio.py:13: UserWarning: Unable to import 'webrtcvad'. This package enables noise removal and is recommended.
  warn("Unable to import 'webrtcvad'. This package enables noise removal and is recommended.")
Arguments:
    enc_model_fpath:   encoder\saved_models\pretrained.pt
    syn_model_dir:     synthesizer\saved_models\logs-pretrained
    voc_model_fpath:   vocoder\saved_models\pretrained\pretrained.pt
    low_mem:           False
    no_sound:          False
    seed:              None

Running a test of your configuration...

Using CPU for inference.

Error: Model files not found. If needed, download them here:
https://github.com/CorentinJ/Real-Time-Voice-Cloning/wiki/Pretrained-models

**Comment by blue-fish** [comment]
@proVoice228 Make sure the files can be found at the locations that the toolbox is expecting.
```
enc_model_fpath: encoder\saved_models\pretrained.pt
syn_model_dir: synthesizer\saved_models\logs-pretrained
voc_model_fpath: vocoder\saved_models\pretrained\pretrained.pt
```

Would appreciate any suggestions for how to make the error message more helpful since someone else had a similar issue in #463.

**Comment by proVoice228** [comment]
![изображение](https://user-images.githubusercontent.com/69386445/89715571-1d29bc00-d9af-11ea-87c6-822384abe293.png)
![изображение](https://user-images.githubusercontent.com/69386445/89715582-32064f80-d9af-11ea-916e-9203d2e95544.png)
![изображение](https://user-images.githubusercontent.com/69386445/89715587-3a5e8a80-d9af-11ea-87d3-7b594bd1cdbc.png)

Have I missed something?

**Comment by proVoice228** [comment]
I've fixed that by changing the relative path to the absolute path

**Comment by blue-fish** [comment]
The toolbox expects python to be run from the Real-Time-Voice-Cloning directory. Otherwise the relative paths will not work.

If this happens to others we may want to determine the absolute path from the location of Real-Time-Voice-Cloning and use that as the default. Thank you for reporting the issue and your solution @proVoice228 .
